ovos_config/mycroft.conf

```diff
@@ -142,20 +142,46 @@
 
   // Intent Pipeline / plugins config
   "intents" : {
+    // this should be removed and True by default IMHO
+    "experimental_ocp_pipeline": true,
+
+    "OCP": {
+      // legacy forces old audio service instead of OCP
+      "legacy": false,
+      // min confidence (0.0 - 1.0) to accept MediaType
+      "classifier_threshold": 0.4,
+      // min conf for each result (0 - 100)
+      "min_score": 50,
+      // filter results from "wrong" MediaType
+      "filter_media": true,
+      // filter results we lack plugins to play
+      "filter_SEI": true,
+      // playback mode
+      // 0 - auto
+      // 10 - audio results only
+      // 20 - video results only
+      "playback_mode": 0,
+      // if MediaType query fails, try Generic query
+      "search_fallback": true,
+      // enable mycroft common play pipeline
+      "legacy_cps": true
+    },
+
     // the pipeline is a ordered set of frameworks to send an utterance too
     // if one of the frameworks fails the next one is used, until an answer is found
     // NOTE: if padatious is not installed, it will be replaced with padacioso (much slower)
     // in the future these will become plugins, and new pipeline stages can be added by end users
     "pipeline": [
         "stop_high",
         "converse",
+        "ocp_high",
         "padatious_high",
         "adapt_high",
+        "ocp_medium",
         "fallback_high",
         "stop_medium",
         "adapt_medium",
         "padatious_medium",
-        "adapt_low",
```

Comment:
without this, "set the volume to ..." or "change the volume to ..." or "turn up/down" does not work.  This should probably be left until the intents are improved with the volume skill